### PR TITLE
Add option to fill missing bam-readcount values with 0s

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

File: vcf_annotation_tools/vcf_readcount_annotator.py [hunk 1]
```diff
@@ -33,6 +34,12 @@ def define_parser():
         help="Path to write the output VCF file. If not provided, the output VCF file will be "
             +"written next to the input VCF file with a .readcount.vcf file ending."
     )
+    parser.add_argument(
+        "-z", "--write-missing-counts-as-zero",
+        help="The readcount fields for any positions that are not found in the bam_readcount_file(s) "
+            +"will be filled with 0s.",
```
Comment (chrisamiller) [inline vcf_annotation_tools/vcf_readcount_annotator.py]: Suggested change in help text:
"assume that positions not found in the bam-readcount file(s) have zero depth.  (This is the default behavior of bam-readcount) These positions will have all their readcount fields set to 0".

**Review by chrisamiller — APPROVED** [review]
Looks good. is `-z` set by default? (I think it should be)

File: vcf_annotation_tools/vcf_readcount_annotator.py (hunk 1 above)
Comment (susannasiebert) [reply vcf_annotation_tools/vcf_readcount_annotator.py]: Right now this is not the default. If we want this behavior to be the default, do we even need this flag?

Comment (malachig) [reply vcf_annotation_tools/vcf_readcount_annotator.py]: Yeah I think this should just happen by default. Not sure when we would want to turn that behavior off. I guess if someone knows they are using an incomplete bam-readcount result, they could turn off this default and get NA’s for the missing values instead. Not sure if it is worth supporting that use case?

**Comment by chrisamiller** [comment]
+1

**Pull request merged**